ajax/libs/leaflet-minimap/package.json

```diff
@@ -0,0 +1,30 @@
+{
+  "name": "leaflet-minimap",
+  "filename": "Control.MiniMap.min.js",
+  "version": "3.3.1",
+  "author": "Norkart AS",
+  "description": "A plugin for Leaflet that provides a minimap in the corner of the map view.",
+  "keywords": [
+    "maps",
+    "leaflet",
+    "client",
+    "minimap"
+  ],
+  "license": "BSD-2-Clause",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/Norkart/Leaflet-MiniMap.git"
+  },
+  "dependencies": {
+    "leaflet": ">=0.7.7"
+  },
+  "npmName": "leaflet-minimap",
+  "npmFileMap": [
+    {
+      "basePath": "dist",
+      "files": [
+        "**/*"
+      ]
```

Comment:
@kennynaoh I think you should also grab `Control.MiniMap.js` and `Control.MiniMap.css` from `src` folder.